src/nvim/eval.c

```diff
@@ -839,13 +839,13 @@ int skip_expr(char_u **pp)
   return eval1(pp, &rettv, FALSE);
 }
 
-/*
- * Top level evaluation function, returning a string.
- * When "convert" is TRUE convert a List into a sequence of lines and convert
- * a Float to a String.
- * Return pointer to allocated memory, or NULL for failure.
- */
-char_u *eval_to_string(char_u *arg, char_u **nextcmd, int convert)
+/// Top level evaluation function, returning a string.
+///
+/// @param convert  when true convert a List into a sequence of lines and convert
+///                 a Float to a String.
+///
+/// @return         pointer to allocated memory, or NULL for failure.
```

Comment:
FYI in case it makes things easier, we do not generally align the content of `@param` and friends. Though I don't have any preference.

Reply:
Doesn't make a difference difficulty-wise, but good to know. Thanks.